### DISCUSSION TITLE
Relocating Zettlr Tutorial from C:\ drive to D:\ drive issue

### DISCUSSION THREAD

**Discussion by bnpwalter** [Zettlr/Zettlr#5434] [General]
I would like to know how to relocate Zettlr Tutorial from my C:\ drive to my D:\ drive.  I copied the tutorial from C to D. Two file instances show in Zettlr.  When I attempt to delete the C:\ drive folder and click OK nothing happens.  So. Closing Zettlr and trying to delete the tutorial using the File Explorer I get a message that "The action can't be completed ..."
![Action can't be completed](https://github.com/user-attachments/assets/a397d779-82cf-4ca0-89d3-ee1af1f944fe)

**Comment by bnpwalter** [comment]
In addition, I cannot delete the new "Zettlr Tutorial" from within Zettlr either.

**Comment by nathanlesage** [comment]
This looks like some issue with your setup. Zettlr does not keep any file locks, so you should be able to just remove the folder while Zettlr is still running and what it should do is just tell you that the folder is gone. There are numerous helper apps that may keep a file locks, however, including Explorer if it wants to index something.
